### ISSUE TITLE
Does rootless podman using slirp4netns allow for LAN access? 

### ISSUE THREAD

**Issue by KCSesh** [containers/podman#10406]
**Description**

I have read this: https://github.com/containers/podman/blob/master/docs/tutorials/basic_networking.md#example-2 and many other network guides for podman.

Something that is still unclear to me is if there is a way to access external hosts on your LAN network. 

The guide linked above shows how container -> container communication works, and it also shows how an external host on the local network can talk to a container web service that is port mapped to the localhost of the the container. 

But is there a route from the container -> external host on the local network using slirp4netns? 

Example: 
So if my LAN network has hostA and hostB, and my container is running on hostA. 
Can my container send requests to host B? 

If the answer is no, which it seems to be from my testing, I am wondering if I could get more insight to why? (since I am able to make requests outside of my network)

Thanks

**Comment by Luap99** [comment]
Yes that should be possible. I tested it on my network and it works. Is it possible that your firewall blocks traffic? Can you connect directly form hostA to hostB?

**Comment by giuseppe** [comment]
slirp4netns runs like a regular process in the host network namespace.

So if an application from hostA can access hostB, it should work.  Unless it is using the same network mask used by the slirp4netns device inside the container network namespace, in that case the traffic won't be forwarded outside

**Comment by KCSesh** [comment]
Looks like it must have been a firewall issue.
I was a bit confused because I couldn't essentially scan for my local network devices from inside the container.
All I see is:

```
root@2632fdcd4c74:/# arp -a
? (10.0.2.3) at MAC1 [ether] on tap0
? (10.0.2.2) at MAC2 [ether] on tap0
```

I don't see my other hosts on my network. 
**I imagine this is expected, are there details to this?** 

However, on a different host, HostC I was able to run:

```
nc -l 192.168.1.90:8001
```

Then in my container on host A, I was able to curl that IP:port and successfully connected. 

My understanding is that from slirp4netns I have no ability to setup network rules so that I could create a rule that would not allow my container to connect to HostB, and only talk to HostC: https://github.com/containers/podman/issues/10297#issuecomment-837552512

That is correct?
